src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -320,6 +320,18 @@ public sealed partial class JsonIgnoreAttribute : System.Text.Json.Serialization
     {
         public JsonIgnoreAttribute() { }
     }
+    [System.AttributeUsageAttribute(System.AttributeTargets.Property, AllowMultiple=false)]
+    public sealed partial class JsonNameAttribute : System.Text.Json.Serialization.JsonAttribute
```

Comment:
I thought we had decided on `JsonPropertyName` everywhere rather than `JsonName`/`Name`?

Reply:
I didn't think we agreed to change it on the attribute since it applied to a property.

Reply:
cc @rynowak

Reply:
Per discussion, renaming to `JsonPropertyName`